docs/src/forge/clean-cache.md

```diff
@@ -0,0 +1 @@
+# snforge clean-cache
```

Comment:
What's the purpose if this?

Reply:
Oh, was created automatically by pycharm I guess, good catch